Testing/functional/e2e/cypress/support/commands.js

```diff
@@ -232,7 +232,6 @@ Cypress.Commands.add("setupDownloads", () => {
 });
 
 Cypress.Commands.add("restoreAuthCookies", () => {
-    globalStorage.authCookies.forEach(cookie => {
-        cy.setCookie(cookie.name, cookie.value);
-    });
+  var names = globalStorage.authCookies.map((x) => x.name);
+  Cypress.Cookies.preserveOnce(...names);
```

Comment:
keep the cookies!

Reply:
This will work only for 2nd level of test, if you chain 3 tests, the 3rd one will fail. Have the functional tests on the timeline all passed locally with this change?

Reply:
Not sure. But if that is the case then the set cookies will have the same problems since at the end cypress will clean the coookies. I can revert it no prob. We will have to look at the cookies again.
https://docs.cypress.io/api/cypress-api/cookies.html#Preserve-Once

Reply:
per my testing last time, preserve-once means that cookie is only preserved one time :-) We can for sure revisit this approach.